test/AutoinstGeneral_test.rb

```diff
@@ -3,58 +3,141 @@
 require_relative "test_helper"
 
 # storage-ng
```

Comment:
If you delete the begin/end, please delete also the `storage-ng` comment. It's meaningless now.